susemanager-utils/testing/docker/scripts/schema_idempotency_test_pgsql.py

```diff
@@ -0,0 +1,225 @@
+#!/usr/bin/python
+import argparse
+import difflib
+import distutils.version as DV
+import json
+import os
+import re
+import shutil
+import subprocess
+import sys
+
+
+def file_exists(path):
+    try:
+        open(path, 'r')
+        return(True)
+    except FileNotFoundError:
+        return(False)
+
+
+def directory_exists(path):
+    if os.path.isdir(path):
+        return(True)
+    return(False)
+
+
+def get_all_files_rpm_pr(prFile):
+    with open(prFile, 'r') as f:
+        changedFiles = []
+        files = json.loads(read(f))['files']
```

Comment:
i would personally split this function for more readablity:

` with open(prFile, 'r') as f:
        changedFiles = []
        files = json.loads(read(f))['files']
`

` with open(prFile, 'r') as f:
        changedFiles = []
        list_of_changed_files_by_pr  = read(f))['files'] // i dunno if is the python way for calling vars
        files = json.loads(list_of_changed_files_by_pr)
`

Reply:
I added @moio suggestion, so the the function is now shorter.